src/cluster.c

```diff
@@ -4510,6 +4510,9 @@ NULL
                 if (clusterBumpConfigEpochWithoutConsensus() == C_OK) {
                     serverLog(LL_WARNING,
                         "configEpoch updated after importing slot %d", slot);
+                    /* After importing this slot, let the other nodes know as
+                     * soon as possible. */
+                    clusterBroadcastPong(CLUSTER_BROADCAST_ALL);
```

Comment:
Not clear why this is in the if statement. We should probably broadcast regardless.

Reply:
You are right, I mess it up that clusterBumpConfigEpochWithoutConsensus will always generate a new config epoch so that clusterUpdateSlotsConfigWith can work.